recipes/hulk/meta.yaml

```diff
@@ -24,4 +25,9 @@ test:
 about:
   home: https://github.com/will-rowe/hulk
   license: MIT
-  summary: Histosketching Using Little Kmers.
+  summary: Histosketching Using Little Kmers
+
+extra:
+  skip-lints:
+    - should_use_compilers
+    - should_be_noarch_generic
```

Comment:
Why have you chosen to not make it `noarch: generic`?

Reply:
I got a bit confused there. Circle error first time was:
```
ERROR: recipes/hulk/meta.yaml:13: should_be_noarch_generic: The recipe should be build as `noarch`
```
I thought I had to change it to should_be_noarch_generic

Do I need to change it as you suggest?

Reply:
Try removing the skip-lints and adding `noarch: generic` instead to the build section. Read more here: https://bioconda.github.io/contributor/linting.html#should-be-noarch-generic

Reply:
Tried this and now failing again. This recipe is using the Go compiler - as this was said to be better than shipping the binaries when I last asked. Should I be just giving binaries now or should I be sticking with compilation?

Reply:
The linter is still unhappy: `ERROR: recipes/hulk/meta.yaml:17: should_use_compilers: The recipe requires a compiler directly`

Are you using go to compile your recipe? Then it should go to the reuquirements/build section - the go compiler I mean.

Reply:
Yes - the go compiler is being used. It's already in the meta as this is just an updated recipe, normally it passes CI fine when I just update the version number and sha256 in my recipes.

Reply:
AFAIK the `bioconda-utils` linter code has changed a bit which is why you may see different behavior than before.
What we need for this recipe is 
1. ```diff
   -  noarch: generic
   ```
    (can't be `noarch` since we have compiled code here)
2. ```diff
   -      - go
   +      - {{ compiler('go') }}
   ```
3. maybe
   ```diff
   -      - mercurial
   ```
   ^-- is `mercurial` really used during the build?

Please apply those changes and then we can see/act upon linter/compile errors we get.

Reply:
thanks! Mercurial was used by one of the Go packages imported by my tool - will see if I still need this or if the package has updated it's import method